src/avgScrabbleScoreFromFile.js

```diff
@@ -11,14 +11,17 @@ let avgScrabbleScore = require('./avgScrabbleScore');
  * sample-words.txt.
  */
 function avgScrabbleScoreFromFile(fileName) {
-  // This is your job. :)
+  let score = 0;
+  let words = fs.readFileSync(fileName, 'utf-8');
+  words = words.split('\n');
```

Comment:
There's a small bug here. Test this with a simple input file that contains something like

```text
aaaa
```